### ISSUE TITLE
does this have a licence

### ISSUE THREAD

**Issue by falconmick** [vallafederico/glsl-gradient-webflow#2]
I've been looking to find a good gradient effect in webGL to customise for my own. It's find if there is no licence because your not giving any permissions, but if it were open to use I would love to :)

**Comment by vallafederico** [comment]
sorry I forgot to add that. do whatever you want with it haha! maybe share when you’re done would love to check that out sent from my iphoneOn 27 Jan 2023, at 13:42, falconmick ***@***.***> wrote:﻿
I've been looking to find a good gradient effect in webGL to customise for my own. It's find if there is no licence because your not giving any permissions, but if it were open to use I would love to :)

**Comment by falconmick** [comment]
I don't think that every piece of code ever made has to be give up free of limitations so there's no need to say sorry, I'm just working on a landing page for a web app I'm building and thought it was neet
